docker-compose.yml

```diff
@@ -41,15 +40,13 @@ services:
         aliases:
           - dm.sbf.engineering
   mysql:
-    image: mysql:8.0
+    image: mysql:5.7
```

Comment:
Why are we reverting to 5.7 here?

Reply:
This is just to try to work through getting the adapter working. This code is all a decade or more old and i'm having difficulty getting the adapter to install. Just trying to minimize the differences while debugging. It may not matter since it's in a container but we'll put it back once we get the rest working.

Reply:
Sounds good, Looks good to me then